tests/lib/tools/snapd-state

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: check-state <jq-filter> <comparison> <expected-res> <error-message>"
+    echo "       print-state <jq-filter>"
+    echo "       change-snap-channel <snap-name> <channel>"
+    echo "       force-autorefresh"
+    echo "       prevent-autorefresh"
+    echo "       wait-for-autorefresh <last-change-id>"
+    echo ""
+    echo "The tool is used to manage the snapd state by editing the state.json file."
+    echo "Also it is used to check state and refresh status"
+}
+
+print_state() {
+    JQ_FILTER=$1
+    if [ -z "$JQ_FILTER" ]; then
+        echo "snapd-state: jq filter is needed"
+        exit 1
+    fi
+    # we are storing jq filter specs in JQFILTER, so we can't use --arg like 
+    # normal with jq, so instead just do a _real_ simple and potentially 
+    # _really_ annoying string concatenation with the actual verbatim variable
```

Comment:
I don't think this comment make sense anymore here as this invocation is much simpler than the original one that was moved here from check_state_json?

Reply:
right, thanks